src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -152,4 +173,15 @@ public bool MapInboundClaims
     /// Defaults to <see cref="ConfigurationManager{OpenIdConnectConfiguration}.DefaultRefreshInterval" />.
     /// </value>
     public TimeSpan RefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultRefreshInterval;
+
+    /// <summary>
+    /// Gets or sets whether <see cref="TokenHandlers"/> or <see cref="SecurityTokenValidators"/> will be used to validate the inbound token.
+    /// </summary>
+    /// <remarks>
+    /// The advantage of using TokenHandlers are:
+    /// <para>There is an Async model.</para>
+    /// <para>The default token handler is a <see cref="JsonWebTokenHandler"/> which is faster than a <see cref="JwtSecurityTokenHandler"/>.</para>
+    /// <para>There is an ability to make use of a Last-Known-Good model for metadata that protects applications when metadata is published with errors.</para>
+    /// </remarks>
```

Comment:
I wonder if we should rethink these doc comments as they might not make full sense now that we've inverted the boolean to be `UseSecurityTokenValidators`.

Maybe we should add why someone would want to use SecurityTokenValidators as well.